gps/_testdata/src/relimport/dotdot/a.go

```diff
@@ -5,7 +5,7 @@
 package dotdot
 
 import (
-	relimport ".."
```

Comment:
This being a test, the use of the named import is helpful to indicate intent. Let's please leave this unchanged.

Reply:
oh.. okay..

Reply:
@sdboyer fixed